fetch/api/cors/resources/corspreflight.js

```diff
@@ -0,0 +1,58 @@
+function headerNames(headers)
+{
+    let names = [];
+    for (let header of headers)
+        names.push(header[0].toLowerCase());
```

Comment:
Nit: This seems unrelated to this PR but maybe make this indent consistent with others in this file?